packages/netlify-cms-core/src/formats/frontmatter.ts

```diff
@@ -3,8 +3,14 @@ import tomlFormatter from './toml';
 import yamlFormatter from './yaml';
 import jsonFormatter from './json';
 
+enum Language {
+  YAML = 'yaml',
+  TOML = 'toml',
+  JSON = 'json',
+}
```

Comment:
I heard that enums is considered to be a bad practice and it's better not to use them

Reply:
Hmm. I would love to understand why (especially in this case).

Reply:
I'm merging this - we can always refactor the code if we find a good reason not to use enums

Reply:
Yeah, in this case it's pretty safe. Problems might happen when you don't explicitly specify indexes. But in that case I don't really see the point of using non-standard language feature when you can easily use standard object literal.

Reply:
> non-standard language feature when you can easily use standard object literal.

Good point. I think it makes it easier to type (instead of doing `keyof typeof <object>`

Reply:
Yeah, the question is as always "is it worth it?" :) ```keyof typeof <object>``` will be just stripped by Babel, while enums will be transpiled into something else.

Reply:
Another good point (which I missed). I did some [reading on this](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums):
![image](https://user-images.githubusercontent.com/26760571/115988053-8ba3be00-a5c0-11eb-8eb8-437264070e13.png)

See PR https://github.com/netlify/netlify-cms/pull/5284